server/@types/documents.d.ts

```diff
@@ -1,6 +1,6 @@
 import { RecallDocument, RescindRecord } from './manage-recalls-api'
 
-type DocumentType = 'document' | 'email' | 'generated'
+type DocumentType = 'document' | 'email' | 'generated' // TODO: PUD-1489: add a new value for documents attached to notes?
```

Comment:
Ar the notes documents going to appear in the recallresponse documents array?

Reply:
For now I think yes.  I'll add the expected Notes presentation and work from there.

Reply:
The notes seem closest to missing document records, as an existing example. Uploads for missing document records don't appear in the main documents array?

Reply:
I was wrong about this, ignore